### ISSUE TITLE
List projects that use WebGL

### ISSUE THREAD

**Issue by w0rm** [elm-community/webgl#7]
I think we need a showcase of the projects that use WebGL. This will help us to raise awareness of the library, overview all the possible use cases, provide more information for the new people.

We can either add a section to the README, or build a small elm app that we publish to http://webgl.elm-community.org.

I can imagine the following sections:

# Examples

Something that may fit in [webgl examples](https://github.com/elm-community/webgl/tree/master/examples), but better be kept outside so we don't have to upgrade them each time.

* [w0rm/elm-webgl-playground](https://github.com/w0rm/elm-webgl-playground)
* [avh4/codevember-2016](https://github.com/avh4/codevember-2016) check [day 10](https://avh4.github.io/codevember-2016/Day10/) and [day 11](https://avh4.github.io/codevember-2016/Day11/)
* [lepoetemaudit/elm-terrain](https://github.com/lepoetemaudit/elm-terrain) a proof-of-concept terrain renderer
* [francisdb/glmorph](https://github.com/francisdb/glmorph) Elm WebGL morphing particle system
* [hendriklammers/month-of-shaders](https://github.com/hendriklammers/month-of-shaders) One GLSL shader for every day of the month August
* [peterszerzo/overeasy.sh](https://github.com/peterszerzo/overeasy.sh/blob/master/Pieces/OurBearingsAreFragile.elm) some generative art, [check the demo here](http://overeasy.sh/our-bearings-are-fragile)

# Libraries

* [elm-2d-game](https://github.com/Zinggi/elm-2d-game)
* [elmo-8](https://github.com/micktwomey/elmo-8)
* [elm-obj-loader](https://github.com/Zinggi/elm-obj-loader)
* [opensolid/scene](https://github.com/opensolid/scene) WIP scene graph package,  [check the demo here](https://opensolid.github.io/examples/lighting.html)

# Projects

* [count-down](https://github.com/psandahl/count-down) 
* [into-the-heavens](https://github.com/nphollon/into-the-heavens) 
* [elm-street-404](https://github.com/zalando/elm-street-404)
* [elm-mogee](https://github.com/w0rm/elm-mogee)
* [elm-snake](https://github.com/tibastral/elm-snake)
* [mage-city](https://github.com/passiomatic/mage-city)
* [kfish/elm-shadertoy](https://github.com/kfish/elm-shadertoy)  incorporate shaders from Shadertoy in Elm projects
* [elm-cubik](https://github.com/w0rm/elm-cubik)

Please mention your project in the comments and I will update this post.

**Comment by Zinggi** [comment]
I really like this idea.
Make sure you don't forget your own stuff :grinning: 
[elm-street-404](https://github.com/zalando/elm-street-404) / [elm-mogee](https://github.com/w0rm/elm-mogee)

**Comment by w0rm** [comment]
I also have my own small webgl playground here: https://github.com/w0rm/elm-webgl-playground

**Comment by francisdb** [comment]
@avh4 has some pretty cool demos
https://github.com/avh4/codevember-2016 (check day 10 and 11)

https://avh4.github.io/codevember-2016/Day10/
https://avh4.github.io/codevember-2016/Day11/

**Comment by w0rm** [comment]
@francisdb thanks! added to the list

**Comment by w0rm** [comment]
Added Elm Terrain by @lepoetemaudit

**Comment by lepoetemaudit** [comment]
@w0rm thanks for the add! I'd better get a move on with the migration to 0.18 so at least it's up-to-date - in progress branch here https://github.com/lepoetemaudit/elm-terrain/tree/dj/elm-0.18

**Comment by Zinggi** [comment]
Something new for the list:
https://zinggi.github.io/randomDemos/webgl/objLoader_modelViewer.html
and
http://package.elm-lang.org/packages/Zinggi/elm-obj-loader/latest

**Comment by w0rm** [comment]
Added [elm-snake](https://github.com/tibastral/elm-snake) to the list

**Comment by passiomatic** [comment]
This is mine: [passiomatic/mage-city](https://github.com/passiomatic/mage-city).

**Comment by francisdb** [comment]
https://github.com/francisdb/glmorph and +1 for creating a page

**Comment by psandahl** [comment]
Simple game, made for fun: https://github.com/psandahl/count-down

**Comment by ianmackenzie** [comment]
Posted on Slack/Twitter a while ago: https://opensolid.github.io/examples/lighting.html

**Comment by w0rm** [comment]
@ianmackenzie yeah, this looks awesome! Can folks already use it? What is the source code we should be linking to?

**Comment by ianmackenzie** [comment]
@w0rm The source code for that example is at https://github.com/opensolid/scene/blob/master/examples/Animated.elm, although I'm afraid I'm still working on that package so it hasn't been published yet! I'll let you know when it is...

**Comment by hendriklammers** [comment]
Last month I wrote 1 GLSL shader (in Elm) every day for learning purposes: https://github.com/hendriklammers/month-of-shaders

**Comment by francisdb** [comment]
@hendriklammers pretty amazing collection of old school demo scene effects you got there :-)

**Comment by peterszerzo** [comment]
Some generative art: http://overeasy.sh/our-bearings-are-fragile (code at https://github.com/peterszerzo/overeasy.sh/blob/master/Pieces/OurBearingsAreFragile.elm)
